packages/smart/contracts/rewards/MasterChef.sol

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is there anything in this file that I need to review?

Reply:
Yep. Most all of it.